src/kOS.Safe/Compilation/KS/Compiler.cs

```diff
@@ -585,6 +586,7 @@ private void IdentifyUserFunctions(ParseNode node)
                 context.UserFunctions.GetUserFunction(funcIdentifier, storageType == StorageModifier.GLOBAL ? (Int16)0 : GetContainingScopeId(node), node);
             int expressionHash = ConcatenateNodes(bodyNode).GetHashCode();
             userFuncObject.GetUserFunctionOpcodes(expressionHash);
+            userFuncObject.IsFunction = !isLock;
```

Comment:
I guess keep it for now.  Now that I think of it, that *is* the meaning of IsFunction.
It means "a real function as opposed to being the kind of pseudo-function that comes from being a lock".